tests/unit/components/frost-radio-group-test.js

```diff
@@ -42,8 +42,8 @@ describeComponent(
 
       expect(
         component.get('inputs'),
-        'inputs: undefined'
-      ).to.be.undefined
+        'inputs: []'
+      ).to.be.eql([])
```

Comment:
Sorry,  I totally misread that. I'm not sure why you'd favor `to.have.length()` over actually making sure it's the right thing. I guess when it's empty it's not a big deal, but when you expect something there, it'd be much preferred to use `.eql()` I'd think.